### ISSUE TITLE
Thick nuchal fold

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7222]
**Preferred term label:**
Thick nuchal fold

**Synonyms**
Increased nuchal fold 

**Definition (free text, please give PubMed ID)**
--

**Parent term (use hpo.jax.org/app)**
New term Abn sonographic fetal neck

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
Definition: The nuchal fold is a normal fold of skin seen at the back of the fetal neck. Increased thickness of the nuchal fold is a soft marker associated with aneuploidy.
Nuchal fold thickness is measured on an axial section through the head at the level of the thalami, cavum septi pellucidi, and cerebellar hemispheres (i.e. in the same plane that is used to assess the posterior fossa structures) An NF measurement greater than 5 mm at 14 to 17+6 weeks of gestation, or 6 mm at 18 to 28 weeks has been associated with a markedly increased risk for Down's syndrome.(https://radiopaedia.org/articles/nuchal-fold?lang=us) PMID:30431620 

Comment: The measurement of nuchal fold (NF) thickness during the second trimester is considered to be one of the most sensitive and specific isolated ultrasound marker for the identification of suspected cases of trisomy 21.  PMID:30431620 (also includes an image) [
(https://obgyn.onlinelibrary.wiley.com/doi/abs/10.1002/pd.412)

May also be seen in other aneuploidies as well as Turner's syndrome, Noonan's syndrome, Roberts syndrome, Zellweger syndrome and with congenital heart disease  and  with pathogenic CNVs

**Comment by pnrobinson** [comment]
Thanks, @patt1morgan, I updated the definition
